examples/diagram-fit-on-load/README.md

```diff
@@ -23,13 +23,12 @@ Javascript example to demonstrate how can fit the BPMN diagram on load.
 
 2. Load BPMN content
 ```javascript
-const containerId = 'bpmn-container';
-const bpmnVisualization = new BpmnVisualization(window.document.getElementById(containerId));
+const bpmnContainerElt = window.document.getElementById('bpmn-container');
```

Comment:
:question: I don't understand why we create an indirection with this variable. Why don't put directly the value in the constructor? If passing the html element is a pain, let's change the constructor to let user pass the element id directly.
We could also remove the constructor from the example as this is not what we want to highlight in this example

Reply:
For me, it is more readable like this ^^

Reply:
Ok, let's keep it as it is proposed: we had the same discussion in https://github.com/process-analytics/bpmn-visualization-js/pull/892 and we have decided to keep the variable for now.